Homework-5/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Homework_5
+{
+    internal class Program
+    {
+        /// <summary>
+        /// Main method to execute the program.
+        /// </summary>
+        /// <param name="args">Command-line arguments.</param>
+        static void Main(string[] args)
+        {
+            List<int> originalList = Enumerable.Range(1, 10).ToList();
+
+            List<int> squaredList = originalList.Select(x => x * x).ToList();
```

Comment:
squaredList, evenNumbersList and sumOfEvenNumbers should have been defined inside local scope.